### PR TITLE
add support tomochain network

### PR DESCRIPTION
**Pull request by khaihkd**

### CODE REVIEW

**Comment by kvhnuke** [comment]
Hi Thank you for the PR
I dont believe, assetsHandler and activityHandlers are implemented properly
https://github.com/enkryptcom/enKrypt/pull/231/files#diff-3b511a8f3badaa9b2a90c64e1f082b2a505f8718e120a0645fcdc94727978017R22-R23

**Comment by khaihkd** [comment]
hello @kvhnuke, sorry for replying late.

We are an EVM chain, we follow this document https://myetherwallet.gitbook.io/enkrypt-documentation/adding-an-evm-network. But this doc, it does not have documents for assetsHandler and activityHandlers. So, what do we need to integrate this function?

**Comment by kvhnuke** [comment]
> hello @kvhnuke, sorry for replying late.
> 
> We are an EVM chain, we follow this document https://myetherwallet.gitbook.io/enkrypt-documentation/adding-an-evm-network. But this doc, it does not have documents for assetsHandler and activityHandlers. So, what do we need to integrate this function?

Also updated the link to help with more information
https://myetherwallet.gitbook.io/enkrypt-documentation/adding-an-evm-network

File: packages/extension/src/providers/ethereum/networks/tomo.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  isTestNetwork: false,
+  currencyName: "TOMO",
+  currencyNameLong: "TomoChain",
+  node: "wss://ws.tomochain.com",
+  icon: require("./icons/tomo.png"),
+  gradient: "#00e8b4",
+  coingeckoID: "tomochain",
+  basePath: "m/44'/889'/0'/0",
+  assetsInfoHandler,
+  activityHandler: wrapActivityHandler(EtherscanActivity),
```
[the remark this answers is not in the document]
Comment (khaihkd) [reply packages/extension/src/providers/ethereum/networks/tomo.ts:23]: > you have to add the etherscan api link here https://github.com/enkryptcom/enKrypt/blob/main/packages/extension/src/providers/ethereum/libs/activity-handlers/providers/etherscan/configs.ts#L3

we updated. could you please check it?

**Comment by khaihkd** [comment]
Thanks @kvhnuke, I update the scan API, could you please check it?

**Comment by khaihkd** [comment]
@kvhnuke it is doc for API. Is it wrong?

**Comment by kvhnuke** [comment]
> @kvhnuke it is doc for API. Is it wrong?

basically you have to implement that api for enkrypt to get past activity for a user
so this window 
image

will work for tomochain users

**Comment by kvhnuke** [comment]
you can use the etherscan handler as a template
https://github.com/enkryptcom/enKrypt/blob/main/packages/extension/src/providers/ethereum/libs/activity-handlers/providers/etherscan/index.ts

**Comment by khaihkd** [comment]
@kvhnuke 
We need an API like that
  `https://tomoscan.io/api?module=account&action=txlist&address=${address}`
And just fill `https://tomoscan.io`
Is that rights?

**Comment by khaihkd** [comment]
Hi @kvhnuke 
We added activity handlers and assets handlers. Could you please check it?

**Pull request merged**